intro-HPC/ch_interactive_gent.tex

```diff
@@ -0,0 +1,54 @@
+\chapter{HPC-UGent interactive cluster}
+\label{ch:interactive_ugent}
+
+
+\section{Purpose}
+\label{sec:interactive_ugent_pupose}
+The purpose of this cluster is to give the user an environment where
+there should be no waiting in the queue to get access to a limited
+number of resources. This environment allows a user to immediatelty
+start doing, and is the ideal place for interactive work such as
+development, debugging and light prodcution (typically sufficient
+for training and/or courses). This enviroment should be seen as an
+extension of the loginnodes, instead of the a compute resource.
```

Comment:
`loginnodes` -> `login nodes`

last part doesn't make sense: `of the a` (missing word?)